File: MODUL-1/Python/PRAK108-2310817220002-RaudatulSholehah.py
print(f"Diketahui : ")
putaran = int(input("Pak Dengklek mengelilingi taman: "))
jarak = int(input("Jarak tempuh Pak Dengklek: "))

jari_jari = jarak / putaran
hasiljari_jari = jari_jari / (2 * 3.14)

print(f"Jawaban : ")
print(f"Jari-jari taman yang dikelilingi Pak Dengklek adalah { hasiljari_jari:.2f}")

File: MODUL-1/C/PRAK108-2310817220002-RaudatulSholehah.c
#include <stdio.h>
int main()
{
    float putaran, jarak;

    printf("Diketahui : \n");
    printf("Pak Dengklek mengelilingi taman: ");
    scanf("%f", &putaran);

    printf("Jarak tempuh Pak Dengklek: ");
    scanf("%f", &jarak);

    float jari_jari = jarak / putaran;
    float hasiljari_jari = jari_jari / (2 * 3.14);

    printf("Jawaban : \n");
    printf("Jari-jari taman yang dikelilingi Pak Dengklek adalah %.2f\n", hasiljari_jari);
    return 0;
}
